src/main/java/seedu/condonery/logic/parser/property/TypePropertyCommandParser.java

```diff
@@ -25,9 +26,22 @@ public TypePropertyCommand parse(String args) throws ParseException {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, TypePropertyCommand.MESSAGE_USAGE));
         }
+        System.out.println(trimmedArgs);
+
+
 
         String[] nameKeywords = trimmedArgs.split("\\s+");
 
+        for (int i = 0; i < nameKeywords.length; i++) {
+            try {
+                ParserUtil.parsePropertyType(
+                        nameKeywords[i]);
+            } catch (IllegalArgumentException ex) {
```

Comment:
If i'm not wrong! currently the parsePropertyType throws ParseException. Maybe instead you don't have to catch illegalArgumentException, and leave it to the parser to handle.

Reply:
thanks, fixed in latest commit